deno.jsonc

```diff
@@ -38,7 +38,6 @@
     "test:update": "deno task test -- -- --update",
     "mdbook:watch": "mdbook watch -o",
     "bench": "deno bench -A --no-check=remote --unstable",
-    "multisig": "deno task run test_util/ctx.ts -- deno task run examples/multisig_transfer.ts",
```

Comment:
Why was this removed?

Reply:
When I committed the multisig example, I forgot to remove this task